python/init.sls

```diff
@@ -1,5 +1,27 @@
 {% from 'python/map.jinja' import python with context %}
 
+# There is a bug in the ubuntu pip package https://bugs.launchpad.net/ubuntu/+source/python-pip/+bug/1306991
+# This command will run if pip is broken and install a version of pip that works!
+pip-fixer:
+  cmd.run:
+    - name: 'python2.7 /tmp/get-pip.py'
+    - unless: pip
+    - require:
+      - file: pip-fixer
+      - cmd: requests
+  file.managed:
+    - name: /tmp/get-pip.py
+    - mode: 700
+    - source: salt://python/files/get-pip.py
+
+requests:
+  cmd.run:
+    - name: pip install requests==2.5.3
+    - unless: pip list | grep "requests (2.5.3)"
```

Comment:
We probably need a comment saying why we're using a cmd.run rather than the pip state here